### ISSUE TITLE
[ISSUE]: Black screen after launching anything from HDD/SSD

### ISSUE THREAD

**Issue by Nuc1eoN** [israpps/FreeMcBoot-Installer#31]
### Checks

- [X] I have checked [__existing issues__](https://github.com/israpps/FreeMcBoot-Installer/issues) for duplicates and found none
- [X] I am using a build downloaded from [this repository](https://israpps.github.io/FreeMcBoot-Installer/test/8_Downloads.html)

### Describe the issue

Hi, I have dedusted my old fat PS2 and trying to get a good setup. So far playing from USB worked pretty well.

Now I've ordered the GameStar SATA adapter and connected my 500GB SSD. I have set up everything _exactly_ according to https://israpps.github.io/FreeMcBoot-Installer/test/FHDB-TUTO.html.

After I am finished, FreeHDBoot loads successfully, but I am getting a black screen as soon as I try to launch anything! E.g. I am getting a black screen after clicking on: both uLauchELF entries, OpenPS2Loader and FreeHDBoot Configurator.

Therefore I have started up FMCB from my memory card. I am entering OPL (works fine from memcard) and select HDD games list. OPL lists the game ISOs on HDD properly. But as soon as I start them I am in the black screen again. Starting from USB is fine. Tried both OPL 1.1 stable and 1.2 beta.

I have then tried to rule out an issue with the SSD. I have installed a 2.5" HDD this time - same behavior.

I have let the black screen wait for 30 minutes but no changes, no message whatsoever.

My PS2 fat is unmodified (no modchip).
What can I do?

### Console model

SCPH-30004

### Where do you have the issue?

FreeHdBoot

### Program version (if possible, also provide hash seen on installer used)

1.966

**Comment by israpps** [comment]
Modchip conflict

**Comment by Nuc1eoN** [comment]
> Modchip conflict

@israpps Like I have written in my bug report, my PS2 is unmodified.

**Comment by Nuc1eoN** [comment]
Tried FHDB v1.965 same results.

TL;DR:

- wLaunchELF KHN HDD image from HDD works (__mbr.raw)
- FHDB installs fine and boots into main menu
- FHDB: Black screen after launching uLauchELF, OPL or FreeHDBoot Configurator from main menu. No further messages or notices.
- FCMB: uLauchELF can read the HDD files fine. Even OPL can see the games installed on HDD, but upon launch -> black screen. Launching USB games works fine.
- (Additionally) I've noticed the reset button on FHCB deos not work (nothing happens on short press). But in FCMB or stock menu it works normally. Not sure if that is expected?

----

I have been 99% percent sure the PS2 is unmodified, because this is my childhood PS2 I bought second hand from a dude. But was never able to play priated games, and also the warranty sticker was undamaged.

But still, I have now opened my PS2 to verify:
![ps2_pcb1](https://github.com/israpps/FreeMcBoot-Installer/assets/2538022/05a8bfda-94bb-4b38-a137-5cbaf9d41ead)
![ps2_pcb2](https://github.com/israpps/FreeMcBoot-Installer/assets/2538022/e36d9c26-0d27-4fdd-8013-aec31e29f5f5)

To me, the PS2 looks unmodified.
It's a stock SCPH-30004 from Sony factory.

**Comment by israpps** [comment]
> * (Additionally) I've noticed the reset button on FHCB does not work (nothing happens on short press). But in FCMB or stock menu it works normally. Not sure if that is expected?

Normal behavior, sony made it this way.


> I have been 99% percent sure the PS2 is unmodified, because this is my childhood PS2 I bought second hand from a dude. But was never able to play pirated games, and also the warranty sticker was undamaged.

Then something is wrong on the program or the console.

Or the problem is the network adapter?
Not sure

**Comment by Nuc1eoN** [comment]
>Or the problem is the network adapter?

I will try to get my hands on an original Sony Adapter. But I will need an PATA to SATA adapter aswell. The bitfunx ones are recommended?

>Try PS2BBL, if it doesn't work, then something quite serious is going on

Ok I will give it a go.

Thanks for your input. Give me some time to set it all up and I will report back.

**Comment by Nuc1eoN** [comment]
I di not get around trying PS2BBL, but this issue got fixed by using an original Sony Network adapter with a bitfunx SATA upgrade.

So the GameStar Network adapter was the cause.
